### PR TITLE
Prometheus

### PR DESCRIPTION
**Pull request by hanazuki**
- [x] prometheus-operator
- [x] prometheus 
- [ ] prometheus rule
- [x] alertmanager
- [ ] alertmanager config
- [x] snmp_exporter
- [x] blackbox-exporter

### CODE REVIEW

File: tf/k8s-prom/kubernetes.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+data "aws_eks_cluster" "rk22" {
```
Comment (sorah) [inline tf/k8s-prom/kubernetes.tf:1]: - https://github.com/ruby-no-kai/rubykaigi-nw/blob/c866cffcabaa3fc91ea02228efc905645ab984f4/tf/dhcp/backend.tf#L9
- https://github.com/ruby-no-kai/rubykaigi-nw/blob/c866cffcabaa3fc91ea02228efc905645ab984f4/tf/dhcp/rds.tf#L44
- https://github.com/cookpad/terraform-aws-eks/blob/4e424f4ecbb848cbf26df2afe22f333f99d10f9a/modules/cluster/outputs.tf

remote state で読んできたほうがらくかも?

File: tf/k8s-prom/kube-prometheus-stack.tf [hunk 2]
```diff
@@ -0,0 +1,65 @@
+resource "helm_release" "kube-prometheus-stack" {
+  repository = "https://prometheus-community.github.io/helm-charts"
+  chart      = "kube-prometheus-stack"
+  version    = "39.9.0"
+
+  name             = "kube-prometheus-stack"
+  namespace        = "monitoring"
```
Comment (sorah) [inline tf/k8s-prom/kube-prometheus-stack.tf:7]: default でいいけど分けたならそれはそれで…

**Review by sorah — APPROVED** [review]
:wafArigatou:

File: tf/k8s-prom/kubernetes.tf (hunk 1 above)
Comment (hanazuki) [reply tf/k8s-prom/kubernetes.tf:1]: かも

**Comment by hanazuki** [comment]
あちこちさわってしまってので一旦まーじ

**Pull request merged**